jenkins/catapult/runDockerBuildInnerBuild.py

```diff
@@ -29,7 +30,16 @@ def _prepare_directory(self, enable_code_coverage, out_dir):
 
 	def _prepare_environment_variables(self):
 		if self.use_conan:
-			self.environment_manager.set_env_var('HOME', '/conan')  # conan cache directory
+			# conan cache directory
+			if not self.environment_manager.is_windows_platform():
+				self.environment_manager.set_env_var('HOME', '/conan')
+			else:
+				self.environment_manager.set_env_var('CONAN_USER_HOME', '/conan')
+				self.environment_manager.set_env_var('CONAN_USER_HOME_SHORT', 'None')
+				# probably should be in the machine image not here
```

Comment:
probably :D

Reply:
Yea, should remove this and just leave it in the image.  Our scripts should not be updating users setting